zkstack_cli/crates/zkstack/src/commands/private_rpc/mod.rs

```diff
@@ -127,15 +131,19 @@ pub async fn init(shell: &Shell) -> anyhow::Result<()> {
     logger::info(msg_private_rpc_docker_image_being_built());
     Cmd::new(cmd!(shell, "docker build -t private-rpc .")).run()?;
 
-    let mut db_config = prompt_db_config(&chain_config)?;
+    let db_config = if args.use_default {
+        let private_rpc_db_name: String = generate_private_rpc_db_name(&chain_config);
+        let private_rpc_db_name = slugify!(&private_rpc_db_name, separator = "_");
```

Comment:
why is it 2 functions? i'd assume it has to be the only. is there are any reason not to slugify it?

Reply:
actually, we may not need it at all, it should be already in correct format at this point, I removed the slugify!

Reply:
no you don't ;)